debugger/src/main/kotlin/org/rust/debugger/settings/RsDebuggerSteppingSettingsConfigurableUi.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.debugger.settings
+
+import com.intellij.openapi.options.ConfigurableUi
+import com.intellij.ui.components.JBCheckBox
+import com.intellij.ui.layout.panel
+import javax.swing.JComponent
+
+class RsDebuggerSteppingSettingsConfigurableUi : ConfigurableUi<RsDebuggerSettings> {
+    private val filterStdlib = JBCheckBox(
+        "Do not step into stdlib (std, core, alloc)",
```

Comment:
Let's move to to bundle like this
```suggestion
RsDebuggerBundle.message("settings.rust.debugger.do.not.step.into.stdlib.checkbox"),
```